k8s-canary-rollout.sh

```diff
@@ -29,31 +28,37 @@ healthcheck(){
 
 cancel(){
     echo "[CANARY] Cancelling rollout - healthcheck failed"
-    
+
     echo "[CANARY SCALE] Restoring original deployment to $PROD_DEPLOYMENT"
     kubectl apply -f $WORKING_VOLUME/original_deployment.yaml -n $NAMESPACE
     kubectl rollout status deployment/$PROD_DEPLOYMENT
 
     #we could also just scale to 0.
-    echo "[CANARY DELETE] Removing canary completely"
+    echo "[CANARY DELETE] Removing canary deployment completely"
     kubectl delete deployment $CANARY_DEPLOYMENT
 
+    echo "[CANARY DELETE] Removing canary horizontal pod autoscaler completely"
+    kubectl delete deployment $CANARY_DEPLOYMENT -n $NAMESPACE
```

Comment:
This doesn't look correct. It seems like the code just the deletes the deployment twice

Reply:
Dohhh! let me fix